pkg/kp/constants.go

```diff
@@ -10,6 +10,7 @@ const (
 	INTENT_TREE  string = "intent"
 	REALITY_TREE string = "reality"
 	HOOK_TREE    string = "hooks"
+	LOCK_TREE    string = "lock"
```

Comment:
yeah. If we could take locks on the intent tree without writing the keys by accident, this new tree would have been unnecessary.